pagespeed/automatic/proxy_interface_test.cc

```diff
@@ -381,6 +381,46 @@ TEST_F(ProxyInterfaceTest, LoggingInfo) {
   EXPECT_TRUE(logging_info()->is_request_disabled());
 }
 
+// Regression test for https://github.com/pagespeed/mod_pagespeed/issues/1553
+TEST_F(ProxyInterfaceTest, TestNoDebugAbortAfterMoreThenOneYear) {
+  GoogleString text;
+  RequestHeaders request_headers;
+  ResponseHeaders response_headers;
+  response_headers.Add(HttpAttributes::kContentType,
+		       kContentTypeHtml.mime_type());
+  response_headers.SetStatusAndReason(HttpStatus::kOK);
+
+  GoogleString url = "http://test.com/flush_subresources.html"
+    "?PageSpeedFilters=+extend_cache_css,-inline_css";
```

Comment:
would prefer setting this in options, e.g. with this pattern elsewhere in the file:

  RewriteOptions* options = server_context()->global_options();
  options->ClearSignatureForTesting();
  options->EnableFilter(RewriteOptions::kExtendCacheCss);
  options->DisableFilter(RewriteOptions::kInlineCss);
  server_context()->ComputeSignature(options);
  
Then you can omit the query-params from the URL, and this will behave more like normal usage.  Admittedly this takes more lines, but I think it's more important to mimic live usage.